api/lib/redisHelper.mjs

```diff
@@ -60,7 +70,15 @@ export async function getStudent(email) {
             { expectedType: 'string', email },
         );
     }
-    return await getEntry(email);
+    let student;
+    try {
+        student = await getEntry(email);
+    } catch (error) {
+        throw new StudentNotFoundError(`Student ${email} is not in the database.`);
```

Comment:
If you want, you can just throw the `KeyNotFound` (or whatever you decide to name the error) error here instead of making a new one for student and then handle it in the API.  Alternatively, you're welcome to make a new error that inherits from `KeyNotFound` that is specific to the student.  Up to you.